rktl_autonomy/src/rktl_autonomy/_ros_interface.py

```diff
@@ -163,83 +169,95 @@ def reset(self):
             msg.hardware_id = self.__LOG_ID
             msg.values = [KeyValue(key=key, value=str(value)) for key, value in info.items()]
             self.__log_pub.publish(msg)
-            # update variables (update time after reset)
+            # increment the episode count
```

Comment:
This is one of those comments that likely states the obvious, we can probably go without a comment here

Reply:
Ye, I should have caught that...